tests/containers/workbenches/workbench_image_test.py

```diff
@@ -89,6 +89,7 @@ def test_ipv6_only(self, image: str, test_frame):
                         host = "localhost"
                         port = podman_machine_utils.find_free_port()
                         socket_path = os.path.realpath(docker_utils.get_socket_path(client.client))
+                        logging.debug(f"{socket_path=}")
```

Comment:
Just curious - are we keeping this on purpose?

Reply:
Yes, otherwise the exception after `podman machine` was not found (that may be printed later) is meaningless. This writes down the socket it was looking for.